### PR TITLE
Master wowl action helpers ref aab

### PR DESCRIPTION
**Pull request by aab-odoo**

### CODE REVIEW

File: addons/web/static/tests/setup.js [hunk 1]
```diff
@@ -37,6 +37,11 @@ function patchBrowserWithCleanup() {
             });
         },
     });
+    // in tests, we never want to interact with the real local/session storages.
+    patchWithCleanup(browser, {
```
Comment (aab-odoo) [inline addons/web/static/tests/setup.js]: merge with other one

**Comment by ged-odoo** [comment]
robodoo delegate=aab-odoo

**Review by ged-odoo — APPROVED** [review]

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Pull request closed without merging**